entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+python manage.py migrate --noinput --settings=metricsmentor.settings_docker
+
+python manage.py runserver --settings=metricsmentor.settings_docker 0.0.0.0:8000
```

Comment:
@ndittren I think is best if you run it with Gunicorn even if is within the docker container.  That's recommended by the Django community.